test/framework/toy_build.py

```diff
@@ -1194,6 +1194,40 @@ def test_toy_extension_patches(self):
 
         self.test_toy_build(ec_file=test_ec)
 
+    def test_toy_extension_git(self):
+        """Test install toy that includes extensions with git_config."""
+        test_ecs = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'easyconfigs', 'test_ecs')
+        toy_ec = os.path.join(test_ecs, 't', 'toy', 'toy-0.0.eb')
+        toy_ec_txt = read_file(toy_ec)
+
+        # Tar-ball which should be created via 'git_config'
+        ext_tgz = 'exts-git.tar.gz'
+        ext_tarball = os.path.join(self.test_sourcepath, 't', 'toy', ext_tgz)
+
+        # Dummy source code required for extensions build_step to pass
+        ext_src = 'int main() { return 0; }'
+        ext_cfile = 'exts-git.c'
+
+        test_ec = os.path.join(self.test_prefix, 'test.eb')
+        test_ec_txt = '\n'.join([
+            toy_ec_txt,
+            'prebuildopts = "echo \\\"%s\\\" > %s && ",' % (ext_src, ext_cfile),
+            'exts_list = [',
+            '   ("exts-git", "0.0", {',
+            '       "buildopts": "&& ls -l %s",' % ext_tarball,
```

Comment:
@kelseymh Maybe (also) use `ls -l a_directory/a_file.txt` here (which should be there, since that's part of `testrepository`)? That can't just come out of thin air, it must have resulted from downloading the git repo and unpacking the resulting tarball?

Reply:
Sure, I can do that!  I figured that since the tarball name was "unexpected", it couldn't have come into existence automatically either.

(I'll "resolve" this once I updated the code.)